application/src/strategy/charts/BarChart.tsx

```diff
@@ -4,22 +4,27 @@ import { Bar } from "react-chartjs-2";
 
 Chart.register(CategoryScale, LinearScale, BarElement);
 
-type DataSet = [Color, Record<string, number>];
+
 type Color = string;
+
+interface DataSet {
```

Comment:
DataSet is not meaningful

Reply:
yes it is, chart uses datasets that you can toggle on and off to show different bars from different dataSets

Reply:
I wouldn't think DataSet would have a color property

Reply:
different datasets need to be differentiated, therefore they need to have different colors.